cpp/src/plasma/tf/test.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import pyarrow as pa
+import pyarrow.plasma as plasma
+import tensorflow as tf
+
+import time
+
+zero_out_module = tf.load_op_library('./plasma_op.so')
+
+client = plasma.connect("/tmp/plasma", "", 64)
```

Comment:
Minor comment, but maybe do a mktemp type thing here instead? I know some folks run tests in parallel on the same machine.